gwpy/testing/utils.py

```diff
@@ -148,35 +148,50 @@ def _assert_quantity(q1, q2, array_assertion=assert_array_equal):
     array_assertion(q1.value, q2.value)
 
 
-def assert_quantity_sub_equal(a, b, *attrs, **kwargs):
-    """Assert that two `~gwpy.types.Array` objects are the same (or almost)
+def assert_quantity_sub_equal(
+    a,
+    b,
+    *attrs,
+    almost_equal=False,
+    exclude=None,
+    **kwargs,
+):
+    """Assert that two `~gwpy.types.Array` objects are the same (or almost).
 
     Parameters
     ----------
     a, b : `~gwpy.types.Array`
-        the arrays two be tested (can be subclasses)
+        The arrays two be tested (can be subclasses).
```

Comment:
Is this a typo?
```suggestion
        The arrays to be tested (can be subclasses).
```

Reply:
Yes, and a six-year-old one at that, good spot.